Docs/Sample.plist

```diff
@@ -1611,6 +1611,150 @@
 				<key>Path</key>
 				<string>Ext4Dxe.efi</string>
 			</dict>
+			<dict>
```

Comment:
In my opinion, only put these drivers in SampleCustom.plist.

Reply:
Yes, I agree, this is not a common set of drivers, I'll remove them from Sample.plist